api/src/application/user/user-controller.js

```diff
@@ -124,15 +126,47 @@ module.exports = (
       }
     }),
 
+    emailVerifySuccessHandler: async (request, reply) => {
+      const token = await emailTokenService.find(request.query.token);
+      const user = await userService.findById(token.get('user_id'));
+
+      const profile = user.get('profile');
+      profile.email_verified = true;
+      await userService.update(user.get('id'), { profile });
+      token.destroy();
```

Comment:
Do you want to await this token.destroy() call?

Reply:
I added an await to this in a later PR